web/src/components/SearchListItem.tsx

```diff
@@ -63,24 +63,26 @@ type SearchListItemProps = {
   thumbnail: string | null
 }
 
-const SearchListItem = ({ title, contentWithoutHtml, query, path, thumbnail }: SearchListItemProps): ReactElement => {
-  const excerpt = getExcerpt(contentWithoutHtml, { query, maxChars: EXCERPT_MAX_CHARS })
+const SearchListItem = React.memo(
```

Comment:
🤔 Does it make sense to wrap this in a `React.memo()`? In what cases are these reused? The list item should change on every new char typed in the query as the highlighting also changes then.

Reply:
This is not correct. Using `memo` skips re-rendering when props are UNchanged, not when they are changed. From the [docs](https://react.dev/reference/react/memo#skipping-re-rendering-when-props-are-unchanged):
> React normally re-renders a component whenever its parent re-renders. With memo, you can create a component that React will not re-render when its parent re-renders so long as its new props are the same as the old props. Such a component is said to be memoized.

Since the props will basically change on every render as the query will change on nearly every render, I think this will not lead to an increased performance. Or am I missing something here?

Reply:
I will see if I can do some sort of profiling to see with and without how it performs.

Reply:
I agree with Steffen. On every input change it will anyway change so...